### ISSUE TITLE
confusing doc about language version available

### ISSUE THREAD

**Issue by ghost** [Shippable/support#4492]
Hi,

## Issue
My subscription is configured to run on a node pool of `ubuntu_16.04` version `v6.7.4`. (This parameter is not configurable as code BTW)
So according to [this doc](http://docs.shippable.com/platform/runtime/machine-image/ami-v674/#python) python is available with versions :
- 2.7.12
- 3.7.0 

So I would guess that the following `shippable.yaml` should works :
```yaml
language: python

python:
    - "3.7.0"                        
    - "2.7.12"

build:
    ci:
        - echo "THE CI !!"
```

But it fails :
```
virtualenv -p /usr/bin/python3.7.0 /root/venv/3.7.0
The path /usr/bin/python3.7.0 (from --python=/usr/bin/python3.7.0) does not exist
```

link to the build : https://app.shippable.com/github/CICDtestlumapps/elTest/runs/9/1/console
 

To make it work and know exactly how available python version are named I had to dig into how you build your images.
[Here](https://github.com/dry-dock/u16pytall/blob/master/version/3_7.sh) you can see that python `3.7.0` is installed under the name `3.7`
[Here](https://github.com/dry-dock/u16pytall/blob/master/version/2_7.sh) you can see that python `2.7.12` is installed under the name `2.7`

So the following  `shippable.yaml` should now works :
```yaml
language: python

python:
    - "3.7"                        
    - "2.7"

build:
    ci:
        - echo "THE CI !!"
```
And it did works : https://app.shippable.com/github/CICDtestlumapps/elTest/runs/11/summary/console

## Reproducing
worker node : `ubuntu_16.04` version `v6.7.4`
`shippable.yaml` : 
```yaml
language: python

python:
    - "3.7.0"                        
    - "2.7.12"

build:
    ci:
        - echo "THE CI !!"
```
It fails.

Same worker
`shippable.yaml` : 
```yaml
language: python

python:
    - "3.7"                        
    - "2.7"

build:
    ci:
        - echo "THE CI !!"
```

## Proposal

Do not simplify the name of things installed in the image. If you install python `2.7.12` it should be available under the version name `2.7.12`.
You can also provide friendly name and aliases but document them ( version `2.7` -> `2.7.12`).

**Comment by ric03uec** [comment]
this will be fixed in the next release (v6.8.4) that comes out in next few days

**Comment by ric03uec** [comment]
this has been fixed in the latest release. The [docs](http://docs.shippable.com/platform/runtime/machine-image/ami-v684/#python) mention the exact python version being installed.
